packages/x-components/src/x-modules/facets-next/events.types.ts

```diff
@@ -0,0 +1,45 @@
+import { Facet } from '@empathyco/x-types';
+import { Filter, EditableNumberRangeFilter } from '@empathyco/x-types-next';
+import { FacetsGroup } from './service/types';
+
+/**
+ * Dictionary of the events of Facets XModule, where each key is the event name, and the
+ * value is the event payload type or `void` if it has no payload.
+ *
+ * @public
+ */
+export interface FacetsNextXEvents {
```

Comment:
I'm seeing that you've removed some to the old events from the `FacetXEvents`, sorry for my lack of knowledge, like `BackendFacetsChanged` or `UserClickedFacetAllFilter: Facet['id']` this is ok right?

Reply:
Apologies accepted 🤣. Yep, that's intentional, there are no longer going to be "front-end" or "backend" facets, there are just going to be groups of facets. About the `UserClickedFacetAllFilter` it corresponds to the `UserClickedFacetAllNextFilter`, because in order to simplify both wiring and the service we decide to change its payload type.